packages/ui-components/package.json

```diff
@@ -53,14 +53,16 @@
 	},
 	"dependencies": {
 		"@floating-ui/dom": "^0.4.0",
-		"css-element-queries": "^1.2.3"
+		"css-element-queries": "^1.2.3",
+		"inputmask": "^5.0.8-beta.17"
```

Comment:
Can't you use a release version instead of a beta version?

Reply:
No, because there is an error on the current release version. See this issue for more information https://github.com/RobinHerbots/Inputmask/issues/2581.